build_version.sh

```diff
@@ -21,16 +21,16 @@ fi
 
 cd $FOLDER
 
-cp ../php.ini php.ini
-cp ../start-cron start-cron
-cp ../composer.sh composer
+cp -f ../php.ini php.ini
+cp -f ../start-cron start-cron
+cp -f ../composer.sh composer
 
 echo "Pull latest php:${PHP_VERSION}-fpm"
 docker pull "php:${PHP_VERSION}-fpm" >> ../build.log
 
 echo "Building exozet/php-fpm:${PHP_VERSION}"
 
-echo "FROM php:${PHP_VERSION}-fpm" >> version-Dockerfile
+echo "FROM php:${PHP_VERSION}-fpm" > version-Dockerfile
```

Comment:
Why this change?

Reply:
aligns this with alpine version and make local rebuild possible, because dockerfile will be create new for each build

Reply:
Should be done for all releases, can you open an issue? Thanks 

Plus an issue for the composer upgrade. Thanks

Reply:
Shall we revert this?

Reply:
and fix this then for all branches separately?